sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -32465,6 +32465,163 @@
+              "showCharCount": false,
+              "showWordCount": false,
+              "properties": {},
+              "allowMultipleMasks": false,
+              "addons": [],
+              "tag": "p",
+              "id": "e09oa3u",
+              "className": "",
+              "tags": []
+            }
+          ],
+          "placeholder": "",
+          "prefix": "",
+          "suffix": "",
+          "multiple": false,
+          "defaultValue": null,
+          "protected": false,
+          "unique": false,
+          "persistent": false,
+          "clearOnHide": false,
+          "refreshOn": "",
+          "redrawOn": "",
+          "dataGridLabel": false,
+          "labelPosition": "top",
+          "description": "",
+          "errorLabel": "",
+          "autofocus": false,
+          "dbIndex": false,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "widget": null,
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "custom": "",
+            "customPrivate": false,
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "allowCalculateOverride": false,
+          "encrypted": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "addons": [],
+          "tree": false,
+          "lazyLoad": false,
+          "id": "eiwesth"
+        },
+        {
+          "title": "Declarations panel",
+          "labelWidth": "",
+          "labelMargin": "",
+          "theme": "default",
+          "tooltip": "",
+          "customClass": "formio-panel-bg-secondary",
+          "collapsible": false,
+          "hidden": false,
+          "hideLabel": true,
+          "disabled": false,
+          "modalEdit": false,
+          "key": "declarationsPanel2",
+          "tags": [],
+          "properties": {},
+          "customConditional": "",
+          "conditional": {
+            "json": "",
+            "show": "true",
+            "when": "howWillYouBeAttendingTheProgram",
+            "eq": "Part Time"
+          },
+          "logic": [],
+          "attributes": {},
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "panel",
+          "label": "Panel",
+          "breadcrumb": "default",
+          "tabindex": "",
+          "input": false,
+          "tableView": false,
+          "components": [
+            {
+              "label": "HTML",
+              "className": "category-header-medium-small pb-2",
+              "attrs": [
+                {
+                  "attr": "",
+                  "value": ""
+                }
+              ],
+              "content": "Declarations",
```

Comment:
I think we can remove this component.

Reply:
I understand the approach to use one component to render the wording conditionally for pt or ft. @dheepak-aot and I had a discussion about this, and it would be best if we use two components to manage the wording. Please let me know if you think the current approach needs modification.

Reply:
Sorry, I see that we have that in the requirements.